bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/exceptions/RuntimeErrors.java

```diff
@@ -127,7 +127,12 @@ public enum RuntimeErrors implements DiagnosticCode {
     INVALID_FRACTION_DIGITS("invalid.fraction.digits", "RUNTIME_0099"),
     INVALID_UTF_8_BYTE_ARRAY_VALUE("invalid.utf8.byte.array.value", "RUNTIME_0100"),
     INCOMPATIBLE_ARGUMENTS("incompatible.arguments", "RUNTIME_0101"),
-    DECIMAL_VALUE_OUT_OF_RANGE("decimal.value.out.of.range", "RUNTIME_0102");
+    DECIMAL_VALUE_OUT_OF_RANGE("decimal.value.out.of.range", "RUNTIME_0102"),
+    NEGATIVE_REGEXP_FIND_INDEX("negative.regexp.match.string.index", "RUNTIME_0103"),
```

Comment:
```suggestion
    NEGATIVE_REGEXP_FIND_INDEX("regexp.match.string.negative.index", "RUNTIME_0103"),
```
In order to have consistency.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39444/commits/a7eb91d3a0dd754d2121774779e1e2d7fbe2b64e.